test/test_prototype_transforms_consistency.py

```diff
@@ -312,7 +312,7 @@ def __init__(
             ArgsKwargs(saturation=(0.8, 0.9)),
             ArgsKwargs(hue=0.3),
             ArgsKwargs(hue=(-0.1, 0.2)),
-            ArgsKwargs(brightness=0.1, contrast=0.4, saturation=0.5, hue=0.6),
+            ArgsKwargs(brightness=0.1, contrast=0.4, saturation=0.5, hue=0.3),
```

Comment:
curious why this is updated?

Reply:
Because `0.6` gives us `hue == (-0.6, 0.6)` which lies outside of the `(-0.5, 0.5)` bounds. This test started failing when I was trying to use these arguments for JIT tests.